stumpy/gpu_stump.py

```diff
@@ -50,8 +50,8 @@ def _compute_and_update_PI_kernel(
 
     Parameters
     ----------
-    i : int
-        Sliding window `i`
+    idx : int
+        The index for sliding window `i`
```

Comment:
I think the description should say `idx`

```suggestion
        The index for sliding window `idx`
```

Reply:
I am leaving it as `i` on purpose